### ISSUE TITLE
[Feature] Ability to create a default config file.

### ISSUE THREAD

**Issue by Quanthir** [yandeu/five-server-vscode#12]
Well, since there are lots of libraries out there to remember config options. Is it possible to add a command to vscode to create a default config file with all config options commented? That way it will be very easy to modify server?

**Comment by yandeu** [comment]
_Have you had a look at [types](https://github.com/yandeu/five-server/blob/main/src/types.ts)?_

Guess it is a good idea. Ask the user to create a config file on first server start? Maybe even a guided setup?

**Comment by Quanthir** [comment]
Yes. Just a item in context menu in explorer (like .editorconfig). a config file with all options that are commented out.

**Comment by yandeu** [comment]
Taking a slot in the context menu just for creating a config file seems a bit overkill.

**Comment by Quanthir** [comment]
Then a `ctrl/cmd+shift+p` menu item?

**Comment by yandeu** [comment]
Or a popup on first page view.

I initially wanted to add a little popup menu to the client side for all settings.

Will have a closer look at it in mid January.
